scripts/components/button-group.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+var VNode = require('virtual-dom').VNode;
+var VText = require('virtual-dom').VText;
+
+/**
+ * Button.
+ *
+ * @param {object} props
+ * @param {string} props.text
+ * @param {function} [props.onClick]
+ * @param {string} [props.style=primary]
+ * @param {string} [props.type=button]
+ * @returns {VNode}
+ */
+function button(props) {
```

Comment:
`button-group` is used to wrap buttons, and `button` should never be used directly. Perhaps move it into its own module to improve clarity.